로그인&회원가입.js

```diff
@@ -0,0 +1,105 @@
+document.addEventListener("DOMContentLoaded", function () {
+  const emailInput = document.getElementById("email");
+  const passwordInput = document.getElementById("password");
+  const verifypasswordInput = document.getElementById("verifypassword");
+  const loginButton = document.getElementById("login_box");
+
+  const emailError = document.getElementById("error_email");
+  const passwordError = document.getElementById("error_password");
+  const verifypasswordError = document.getElementById("error_verifypassword");
+
+  // 이메일검증 이벤트
+  if (emailInput) {
+    emailInput.addEventListener("focusout", function () {
+      const emilStatus = emailInput.value.trim();
+      if (emilStatus === "") {
+        showError(emailInput, emailError, "이메일을 입력해주세요.");
+      } else if (!emailValidate(emilStatus)) {
+        showError(emailInput, emailError, "잘못된 이메일 형식입니다.");
+      } else {
+        hideError(emailInput, emailError);
+      }
+      transLoginButton();
+    });
+  }
+  // 비밀번호 검증 이벤트
+  if (passwordInput) {
+    passwordInput.addEventListener("focusout", function () {
+      const passwordStatus = passwordInput.value.trim();
+      if (passwordStatus === "") {
+        showError(passwordInput, passwordError, "비밀번호를 입력해주세요.");
+      } else if (passwordStatus.length < 8) {
+        showError(
+          passwordInput,
+          passwordError,
+          "비밀번호를 8자리 이상으로 입력해주세요."
+        );
+      } else {
+        hideError(passwordInput, passwordError);
+      }
+      transLoginButton();
+    });
+  }
+  // 비밀번호 확인 검증 이벤트
+  if (verifypasswordInput) {
+    verifypasswordInput.addEventListener("focusout", function () {
+      const verifypasswordStatus = verifypasswordInput.value.trim();
+      const passwordStatus = passwordInput.value.trim();
+
+      if (verifypasswordStatus.length < 8) {
+        showError(
+          verifypasswordInput,
+          verifypasswordError,
+          "비밀번호를 8자리 이상으로 입력해주세요."
+        );
+      } else if (verifypasswordStatus !== passwordStatus) {
+        showError(
+          verifypasswordInput,
+          verifypasswordError,
+          "비밀번호가 일치하지 않습니다."
+        );
+      } else {
+        hideError(verifypasswordInput, verifypasswordError);
+      }
+      transLoginButton();
+    });
+  }
+  // 로그인버튼 활성/비활성
+  function transLoginButton() {
+    const emilStatus = emailInput.value.trim();
+    const passwordStatus = passwordInput.value.trim();
+    const verifypasswordStatus = verifypasswordInput.value.trim();
+    if (
+      emilStatus !== "" &&
+      emailValidate(emilStatus) &&
+      passwordStatus !== "" &&
+      passwordStatus.length >= 8 &&
+      verifypasswordStatus === passwordStatus
+    ) {
+      loginButton.classList.remove("disabled");
```

Comment:
P3:
이렇게 if 문의 조건이 길때는 해당 조건을 변수에 저장하시면 조금 더 가독성에 좋습니다.

```suggestion
    const isValidForm = emilStatus !== "" &&
      emailValidate(emilStatus) &&
      passwordStatus !== "" &&
      passwordStatus.length >= 8 &&
      verifypasswordStatus === passwordStatus;
      
    if ( isValidForm ) {
        loginButton.classList.remove("disabled");
```